app/src/components/features/mocksV2/MockEditorIndex/Editor/index.tsx

```diff
@@ -173,8 +173,8 @@ const MockEditor: React.FC<Props> = ({
     return false;
   };
 
-  const handleOnSave = () => {
-    const finalMockData = createMockEditorData();
+  const handleOnSave = (metaData?: Partial<MockEditorDataSchema>) => {
+    const finalMockData = createMockEditorData(metaData);
```

Comment:
Instead of passing metaData to `createMockEditorData`, we can directly use `password` state variable in `createMockEditorData` directly.